docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,26 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting the server icon URI
```

Comment:
Thank you very much for incorporating the comments.

It is recommended to avoid article in the heading (only in heading).
Please fix this and also fix the cross-reference in line number 24, when you change the heading.

Reply:
done